cmd/object-handlers-common.go

```diff
@@ -319,6 +333,18 @@ func checkPreconditions(ctx context.Context, w http.ResponseWriter, r *http.Requ
 	return false
 }
 
+// checkIfUnmodifiedSince - return true if object was modified after(Including critical values) ifUnmodifiedSinceHeader time
```

Comment:
yeah but critical doesn't mean correct in this context.

Reply:
Is this the incorrect reason？
“The Date-Modified header truncates sub-second precision, so use mtime < t+1s instead of mtime <= t to check for unmodified.”

Reply:
thats why we had 1s leeway